### PR TITLE
INTMDB-454: Add support for root query to get org_id from API Key used in Terraform

### PR DESCRIPTION
**Pull request by martinstibbe**
## Description

INTMDB-454: Add support for root query to get org_id from API Key used in Terraform

Link to any related issue(s): 

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: mongodbatlas/root.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+// RootServiceOp handles communication with the APIKey related methods
+// of the MongoDB Atlas API.
+type RootServiceOp service
+
+/*type Root struct {
+	Links      []*Link  `json:"links"`
+	Results    []APIKey `json:"results,omitempty"`
+	TotalCount int      `json:"totalCount"`
+}*/
```
Comment (gssbzn) [inline mongodbatlas/root.go:40]:
```suggestion
```

if not needed

Comment (martinstibbe) [reply mongodbatlas/root.go]: Removed commented code

**Review by gssbzn — DISMISSED** [review]
LGTM if you manually tested this since it seems to be different from the public docs so assuming is a mistake on docs and not on code

**Comment by themantissa** [comment]
@gssbzn if the docs don't include I'd like to get them corrected. Did you check old and new docs?  I can point a docs ticket to this PR to get it fixed if needed.

**Comment by martinstibbe** [comment]
> LGTM if you manually tested this since it seems to be different from the public docs so assuming is a mistake on docs and not on code

Yes legacy Docs are incorrect for returned json value for GET ... I am using this code for TF resource working fine

**Comment by gssbzn** [comment]
@martinstibbe and are the new docs correct https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Root/operation/getSystemStatus?

**Comment by gssbzn** [comment]
could we link to the new docs instead if they are correct?

**Comment by martinstibbe** [comment]
> @martinstibbe and are the new docs correct https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Root/operation/getSystemStatus?

Yes these are accurate I will change link(s) in code

**Review by gssbzn — APPROVED** [review]
LGTM thanks for the update on links

**Pull request merged**